data/json/mutations/mutations.json

```diff
@@ -3854,21 +3830,19 @@
     "description": "You have claws on the ends of your fingers, and can extend or retract them as desired.  Gloves will still get in the way, though.",
     "types": [ "CLAWS" ],
     "prereqs": [ "CLAWS" ],
+    "activated_eocs": [ "EOC_CLAWS_RETRACT" ],
     "category": [ "FELINE" ],
-    "transform": { "target": "CLAWS_RETRACT_active", "msg_transform": "You extend your claws.", "active": false, "moves": 10 },
     "cost": 0
   },
   {
     "type": "mutation",
     "id": "CLAWS_RETRACT_active",
     "name": { "str": "Extended Claws" },
-    "copy-from": "CLAWS_RETRACT",
```

Comment:
Ditto on the deletion in existing saves, and I assume elsewhere (this is not a comprehensive review)